data/brands/amenity/bicycle_repair_station.json

```diff
@@ -0,0 +1,32 @@
+{
+  "properties": {
+    "path": "brands/amenity/bicycle_repair_station",
+    "skipCollection": true
+  },
+  "items": [
+    {
+      "displayName": "RadKULTUR",
+      "id": "radkultur",
+      "locationSet": {"include": ["de"]},
+      "tags": {
+        "amenity": "bicycle_repair_station",
+        "access": "yes",
+        "brand": "RadKULTUR",
+        "brand:wikidata": "Q132562116",
+        "email": "kontakt@radkultur-bw.de",
+        "opening_hours": "24/7",
+        "name": "RadService-Punkt",
+        "manometer": "yes",
+        "manual": "yes",
+        "service:bicycle:chain_tool": "no",
+        "service:bicycle:diy": "yes",
+        "service:bicycle:pump:operational_status": "operational",
```

Comment:
probably something that should be reviewed for each pump, not suggested for all of them
```suggestion
```